alpine-consul/root/etc/services.d/consul-template/run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/with-contenv sh
```

Comment:
The best way to handle this stuff is through `s6-rc` which is brand new. You can view this issue on s6-overly for more information https://github.com/just-containers/s6-overlay/issues/112. Until then, I think we should keep it really simple and run with the loop. At such a time s6-rc is integrate into s6-overlay, then we can refactor.